adr/structure_classes.md

```diff
@@ -0,0 +1,28 @@
+# Structure of Classes Beyond Standard RailsMVC
```

Comment:
Do you want to have the header/format of the other projects that have ADRs?  e.g., https://github.com/pulibrary/figgy/blob/master/architecture-decisions/0001-document-architecture-decisions.md

Reply:
Sure!  @escowles I'm a bit lost on the consequences section.  What would you expect to see there for a document like this?

Reply:
I've sometimes struggled with the consequences section in other ADRs.  For this code organization, I think it's probably something along the lines of:
* This organization makes sense to us, but it's a little different from Rails standard, and a little more like Figgy
* Devs who are familiar with these patterns will understand the division and patterns being used
* Devs who aren't familiar with `change_sets` or `services` might take more time to find things